test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetMvc5Tests.cs

```diff
@@ -2,15 +2,14 @@
 #pragma warning disable SA1402 // File may only contain a single class
 #pragma warning disable SA1649 // File name must match first type name
 
-using System.Collections.Generic;
 using System.Net;
 using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
-    [Collection("IisTests")]
+    [Collection(nameof(IisFixture))]
```

Comment:
The MVC 4 tests use `[Collection("IisTests")]` so these will have a different collection name. Was that intentional?

Reply:
Oops, no, that was something else

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1246/commits/7d3b76f5a67a58d6c14e758097cb28eae2f4b333